### PR TITLE
DSA 1.2.2.5 Algorithms: Recurrence Relations Video

### LINKED ISSUE #486: DSA 1.2.2.5 Algorithms: Recurrence Relations
## Description
How are recurrences solved using Master Method? Discuss the approach, applications and the efficiency.

## Task
- [ ] Documentation
- [ ] Audio
- [ ] Video

## Note
- While requesting the issue to be assigned, please mention your **NAME & BATCH NUMBER**.
- Kindly refer to the [Issue Guidelines](https://github.com/girlscript/winter-of-contributing/blob/DSA/DSA/CONTRIBUTING.md#issues).
- In case you see that the issue has already been assigned, try to request for a different task within the same issue.
- Changes should be made inside the ````DSA/directory```` & ````DSA```` branch.
- Name of the file should be same as the issue.
- This issue is only for 'GWOC' contributors of 'DSA' Domain.

### PR DESCRIPTION
**Pull request by likitarai1**
<hr>

## Description 📜

Created Master Method video
Fixes #486 

<hr>

## Type of change 📝

- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)

<hr>

## Domain of Contribution 📊

- [x] DSA
<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

### CODE REVIEW

**Comment by nimisha-yadav** [comment]
Please review @harshsri28 @sneha-nitdgp

File: DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.5_Master_Method_Video.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Master Method
+
+This method is used to solve recurrence relations of the form:
+
+_T(n) = aT(n/b) + $\theta$ ($n^k log^pn$) where a $\geq$ 1, b > 1, k $\geq$ 0 and p $\in$ R_
```
Comment (harshsri28) [inline DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.5_Master_Method_Video.md:5]: Fix this $\theta$ ($n^k log^pn$) where a $\geq$ 1, b > 1  and  rest is looking good.

**Review by harshsri28 — CHANGES_REQUESTED** [review]

File: DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.5_Master_Method_Video.md (hunk 1 above)
Comment (likitarai1) [reply DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.5_Master_Method_Video.md:5]: The output looks like this - 
recur
Won't this be easier for readers to read?

**Review by sneha-nitdgp — APPROVED** [review]
LGTM

**Review by harshsri28 — APPROVED** [review]
LGTM

**Comment by Muskan-creator** [comment]
Thanks for Contributing @likitarai1 🙌

**Pull request merged**